src/modules/Auth/PasswordSetupSuccess/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+import withTheme from 'components/shared/withTheme';
+import FlowerSuccessSvg from 'assets/svgs/FlowerSuccessSvg';
+import FileSvg from 'assets/svgs/FileSvg';
+import DownloadSvg from 'assets/svgs/DownloadSvg';
+import SuccessScreen from '../components/success';
+import getStyles from './styles';
+
+const PasswordSetupSuccess = ({ styles }) => <SuccessScreen
+  Illustration={FlowerSuccessSvg}
```

Comment:
I think the component can be instantiated here so that the prop name has a lowercase like `illustration`

```suggestion
  illustration={<FlowerSuccessSvg />}
```